boilerplates/hattip/files/hattip-entry.ts

```diff
@@ -57,6 +62,49 @@ if (BATI.has("trpc")) {
   });
 }
 
+if (BATI.has("ts-rest")) {
+  /**
+   * ts-rest route
+   *
+   * @link {@see https://ts-rest.com/docs/serverless/fetch-runtimes/}
+   **/
+  const tsrRouter = tsr.platformContext<{ context: AdapterRequestContext }>().router(contract, {
+    demo: async () => {
+      return {
+        status: 200,
+        body: {
+          demo: true,
+        },
+      };
+    },
+    createTodo: async ({ body }) => {
+      if (BATI.has("drizzle")) {
+        await db.insert(todoTable).values({ text: body.text });
+      } else {
+        lowDb.update(({ todo }) => todo.push({ text: body.text }));
+      }
+      return {
+        status: 200,
+        body: {
+          status: "Ok",
+        },
+      };
+    },
+  });
+
+  router.use("/api/*", (context) => {
+    return tsrFetchRequestHandler({
+      request: context.request,
+      contract,
+      router: tsrRouter,
+      platformContext: {
+        context,
+      },
+      options: {},
+    });
+  });
```

Comment:
Seeing how everything is duplicated even with dedicated adapters, I'm more inclined to use `@ts-rest/serverless/fetch` for all servers.
`tsrRouter` and the generic handler could be extracted to a dedicated `files/servers/ts-rest-handler.ts`.